### ISSUE TITLE
More custom border styles

### ISSUE THREAD

**Issue by lucamaral** [handsontable/handsontable#5245]
Is border styles like double/dashed planned for next releases?

![image](https://user-images.githubusercontent.com/3779622/42509606-d7838732-8422-11e8-92f1-38c328ad7684.png)

**Comment by AMBudnik** [comment]
Cool idea! Thanks for sharing @lucamaral

**Comment by AMBudnik** [comment]
We're still fixing some issue related to the borders but after that, it would be cool to add something new.

not sure if a double border would work well here (especially if you would place it only for a single side of a cell) but dashed one shouldn't be hard to add.

**Comment by lucamaral** [comment]
Nice!
Let me know when something new will be released.

**Comment by AMBudnik** [comment]
It would be easier to track the progress while having this topic opened :)

**Comment by AMBudnik** [comment]
This feature request is closed. It will be automatically reopened after we attach it to a milestone.
Thank you very much for share it. I hope that we'll be able to publish it soon.

**Comment by pnowak** [comment]
We resigned from background-color for borders divs to border-style and thanks to this it is easy to change border style. But when style is dotted or dashed we should remove gray cell borders.

We need to rethink the way we're adding customBorders. 
So far we used 4 divs for every direction - top, right, bottom and left and additionally one div for corner. And all of this divs when was added to DOM doesn't disappear/removed even if cell connected with those borders were scrolled outside the view.

The second problem is that borders are overlaping neighbouring cells. In this PR all borders goes to inside of a cell so borders from neighbouring cells doesn't overlap each other. But if we want to replicate Excel (or Google Sheet) way, in which bottom and right borders will rewrite and shadow top and left borders of neighbouring cells, we need to operate only on two borders - top and left. 
Or maybe we should try with only one div inside a cell. 
Nevertheless customBorders propably should go to cellPropierties.

Last but not least - animated borders after copy action should behave exactly like fill or area - we should add div to DOM only once at initialization with dashed style and blue color. Keep it separated from customBorders, beacuse otherwise copy action will overwrite customBorders.
The biggest problem is the speed of animation depending on selected area. 
But sure - after investigation - this topic isn't connected to customBorders.
